docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -375,21 +333,21 @@ openstack endpoint list | grep manila
 . Test the health of the service:
 +
 ----
-openstack share service list
-openstack share pool list --detail
+$ openstack share service list
+$ openstack share pool list --detail
 ----
 
-. Check on existing workloads:
+. Check existing workloads:
 +
 ----
-openstack share list
-openstack share snapshot list
+$ openstack share list
+$ openstack share snapshot list
 ----
 
 . You can create further resources:
```

Comment:
Does this mean that customers should check whether they are able to create further resources? If not, what is the reason for creating these specific resources?

Reply:
Ack, I think we do not want to ask customers to create additional resources in this context, so I think we can drop this section of put it under an if statement to keep it upstream only. 
@klgill is it ok to keep it upstream only with an `if` here?

Reply:
would like to suggest keeping this.. since, with Manila, we're integrating a storage backend that's external to the cloud, creating new resources will ensure that integration is working fine.. Maybe we can add delete steps so we can clean things up... 

```
$ openstack share delete myshareclone
$ openstack share delete mynfsshare
```


I'd also want to tack on a couple of other steps:

```
$ openstack share access create mysharesnap cephx testuser
$ openstack share access create mynfsshare ip 203.0.113.10
$ openstack share access list mysharesnap
$ openstack share access list mynfsshare
```

These would create access rules that truly "export" a share that's created with the 'create' statements. When using Ceph-NFS, if these steps succeed, it means that the NFS service is working as expected.